src/style/palette.scss

```diff
@@ -1,17 +1,119 @@
-$red: red !default;
-$tomato: tomato !default;
-$orange: orange !default;
-$green: green !default;
-$blue: blue !default;
-$yellow: yellow !default;
+// color coding
+$darkindigo: #401694 !default;
+$blackish: #333333 !default;
+$navy: #225091 !default;
+$american-gray: #808080 !default;
+$aquamarine: #4eccc6 !default;
+$explosive: #c4c4c4 !default;
+$green: #00c875 !default;
+$winter: #9aadbd !default;
+$grass-green: #037f4c !default;
+$river: #68a1bd !default;
+$bright-green: #9cd326 !default;
+$darkblue: #0086c0 !default;
+$saladish: #cab641 !default;
+$lightblue: #579bfc !default;
+$egg-yolk: #ffcc00 !default;
+$chillblue: #66ccff !default;
+$orange: #fdab3d !default;
+$indigo: #5559df !default;
+$peach: #ffadad !default;
+$darkpurple: #784bd1 !default;
+$sunset: #ff7575 !default;
+$berry: #7e3b8a !default;
+$dark-orange: #ff642e !default;
+$purple: #a25ddc !default;
+$red: #e2445c !default;
+$sofia-pink: #ff158a !default;
+$dark-red: #ba3254 !default;
+$lipstick: #ff5ac4 !default;
+$brown: #7f5347 !default;
+$bubble: #faa1f1 !default;
 
-// :root{
-//   @each $key, $value in $colors {
-//       --#{$key} : #{$value};
-//   }
-// }
+// greyscale palette
+$dark: #272a31 !default;
+$wolf-grey: #d4d4d4 !default;
+$ui-grey: #e5e5e5 !default;
+$riverstone: #f5f5f5 !default;
+$white: #ffffff !default;
 
-// primary
-$primary: $red;
-$primary-hover: $tomato;
-$primary-active: $orange;
+// main ui colors
+$primary: #f3e060 !default;
+$secondary: #393c49 !default;
+$tertiary: #eb6165 !default;
+$success: #258750 !default;
+$danger: #d83a52 !default;
+$warning: #d8663a !default;
+
+$color-modifier-percentage: 9% !default;
+
+$primary-hover: darken($primary, $color-modifier-percentage);
+$secondary-hover: darken($secondary, $color-modifier-percentage);
+$tertiary-hover: darken($tertiary, $color-modifier-percentage);
+$success-hover: darken($success, $color-modifier-percentage);
+$danger-hover: darken($danger, $color-modifier-percentage);
+$warning-hover: darken($warning, $color-modifier-percentage);
+
+$primary-active: lighten($primary, $color-modifier-percentage);
+$secondary-active: lighten($secondary, $color-modifier-percentage);
+$tertiary-active: lighten($tertiary, $color-modifier-percentage);
+$success-active: lighten($success, $color-modifier-percentage);
+$danger-active: lighten($danger, $color-modifier-percentage);
+$warning-active: lighten($warning, $color-modifier-percentage);
+
+// We export the variables to show the color codes in the story
+:export {
```

Comment:
remove this export. Remember we don't want to export things that won't be used

Reply:
It is being used, to show the color codes in the stories

Reply:
you should create classes for this. The same way you did for spacings and sizes